plugin/api.go

```diff
@@ -293,6 +293,9 @@ type API interface {
 	// Minimum server version: 5.6
 	GetPostsForChannel(channelId string, page, perPage int) (*model.PostList, *model.AppError)
 
+	// GetTeamStats gets team statistics
```

Comment:
@hanzei, at this point this would be 5.8, right?
```suggestion
	// GetTeamStats gets a team's statistics
	//
	// Minimum server version: 5.8	
```

Reply:
Good catch, your right